newdle/api.py

```diff
@@ -399,6 +400,9 @@ def send_result_emails(code):
     date = newdle.final_dt.strftime('%-d %B %Y')
     start_time = newdle.final_dt.strftime('%H:%M')
     end_time = (newdle.final_dt + newdle.duration).strftime('%H:%M')
+    event = create_calendar_event(newdle)
```

Comment:
i'd rename this to `ics` or `ical_data` - it's a string, not an ical Event object

Reply:
Changed to `ical_data`.